apps/research-service/src/polygon-clipping/polygon-clipping.controller.ts

```diff
@@ -196,6 +196,63 @@ export class PolygonClippingController {
       );
     }
 
+    const sitePolygons = await SitePolygon.findAll({
+      where: {
+        polygonUuid: fixablePolygons,
+        isActive: true
+      },
+      attributes: ["siteUuid"]
+    });
+
+    let entityId: number | undefined;
+    let entityType: string | undefined;
+    let entityName: string;
+
+    if (sitePolygons.length > 0) {
+      const uniqueSiteUuids = Array.from(new Set(sitePolygons.map(sp => sp.siteUuid).filter(uuid => uuid != null)));
```

Comment:
:art: This can use `.filter(isNotNull)`, and in general I prefer to use lodash utilities when they're available:
```
const uniquSiteUuids = uniq(sitePolygons.map(({ siteUuid }) => siteUuid).filter(isNotNull));
```